### ISSUE TITLE
/api/config is missing

### ISSUE THREAD

**Issue by ArndBrugman** [SteveyO/Hue-Emulator#24]
A normal bridge would return configuration on an api call to /api/config even without having a username supplied or being whitelisted. Would be great if you can add this too :+1:

**Comment by SteveyO** [comment]
Sure,  I thought I had this but if not I will include in next release.

**Comment by ArndBrugman** [comment]
Note that this is without the whitelisting. I don't know if this is a regular api call, but it works as expected. It is (whithout the whitelisting) undocumented. [bridgeip]/api/config, not [bridgeip]/api/[whitelistedusername]/config.

**Comment by SteveyO** [comment]
Yes I know.  Actually I wrote the documentation for this myself, about 2 months ago,  although it's hidden a little in the below URL:.
http://www.developers.meethue.com/documentation/message-structure-and-response-0#nonValidUsers

Also, I have just added a link to this from:
http://www.developers.meethue.com/documentation/core-concepts
(Under the "The hue username" heading) to make it a little more obvious.
Steve

**Comment by ArndBrugman** [comment]
Yep! Exactly. Thx :)

**Comment by SteveyO** [comment]
Committed to github.  As per your other issueI will close this when I release a new version.
Note that some fields are missing (e.g. apiversion, swversion, replacesbridgeid, factorynew).  None of these are likely to be added to the Emulator in the future as they are not relevant.

**Comment by mynote** [comment]
zigbeechannel is missing which is important ;-)
